### DISCUSSION TITLE
Feed Submission issues (Submitting ContentType.TXT files)

### DISCUSSION THREAD

**Discussion by smithburger** [abuzuhri/Amazon-SP-API-CSharp#433] [General]
`var feedID = amazonConnectionHHJ.Feed.SubmitFeed("xxx.txt", FeedType.POST_INVENTORY_AVAILABILITY_DATA, new List<string>() {  MarketPlace.US.ID }, null, ContentType.TXT);`

I am trying to submit an inventory feed. The call returns a valid feed id. Two issues.

1) When I call GetFeed the ResultFeedDocumentId is null so I get an "Access to requested resource is denied" error. 

2) When I check the feed status using the MWS scratchpad I can see the feed was submitted as XML despite the ContentType.TXT flag set. So obviously that would return an error and probably results in the above error.

I checked out FeedService and I am not seeing the issue. I am using your example code to test uploading a txt file to update inventory with one line on it.

**Comment by RenzoF** [comment]
Hi POST_INVENTORY_AVAILABILITY_DATA is expecting an XML file, the ones that are expecting a flat file contain FLAT_FILE in the name eg: POST_FLAT_FILE_PRICEANDQUANTITYONLY_UPDATE_DATA

**Reply by smithburger** [reply]
You are correct. My apologies. Worked great.

**Reply by malafalx91** [reply]
dove trovo l'esempio? Qualcuno può dirmi come aggiornare l'inventario tramite txt?
